AdobeStockAsset/Test/_files/creator_rollback.php

```
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\AdobeStockAssetApi\Api\CreatorRepositoryInterface;
+
+$objectManager = Bootstrap::getObjectManager();
+
+try {
+    /** @var CreatorRepositoryInterface $creatorRepository */
+    $creatorRepository = $objectManager->get(CreatorRepositoryInterface::class);
+    $creatorRepository->deleteById(42);
+} catch (\Exception $exception) {
+
```

I don't think it's a good idea to suppress exceptions in the tests